mockserver_test.go

```diff
@@ -256,6 +257,11 @@ func (s *server) handle(conn net.Conn) {
 		l := sc.Text()
 		parts := strings.Split(l, " ")
 		cmd := strings.TrimSpace(parts[0])
+		// to support server commands with specific optional parameters,
```

Comment:
Small tweak, sentences are good
```suggestion
		// Support server commands with specific optional parameters,
```

Reply:
Should be edited in the new commit.